.rubocop.yml

```diff
@@ -16,6 +23,15 @@ Style/SpaceBeforeFirstArg:
 Style/BracesAroundHashParameters:
   Enabled: false
 
+Style/CaseEquality:
+  Enabled: false
```

Comment:
Disabled this one so that we can use `===` in the custom matcher to match strings or regexes without worrying what type it is.